src/Extensions/AzureBlobPayloads/AzureBlobPayloads.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Let's target `net8.0` for this project. Targeting `netstandard2.0` puts a lot of restrictions on what we're able to do in terms of C# features and dependencies.